python/jupytercad_core/src/stlplugin/commands.ts

```diff
@@ -0,0 +1,198 @@
+import { FormDialog } from '@jupytercad/base';
```

Comment:
Let's move these new commands in the base commands?

The `stlplugin` is to allow JupyterCAD to open STL files with the JupyterCAD viewer, it's probably not the right place to put commands for general CAD files.